packages/component-library/.storybook/preview.ts

```diff
@@ -54,6 +55,12 @@ const preview: Preview = {
         { name: "dark", value: DARK_THEME_BACKGROUND_VALUE },
       ],
     },
+    options: {
+      storySort: (a, b) => {
+        // Sort the stories by their id alphabetically
+        return a.id === b.id ? 0 : a.id.localeCompare(b.id, undefined, { numeric: true });
+      },
+    },
```

Comment:
Is this adding much benefit? I guess the default behaviour is that it has the same order as the stories in the code. I personally prefer that because the third story in the sidebar is also the third story in the file

Reply:
I thought it was easier to find the correct story when you have a alphabetical order

Reply:
Ok, I would just use the default behaviour of Storybook. You can still use search function if you can't find something.

Reply:
@Haberkamp Just FYI:
<img width="271" alt="Bildschirmfoto 2024-12-20 um 08 49 02" src="https://github.com/user-attachments/assets/069281db-01fb-4859-80cc-ced47f30c222" />

This is how it looks without sorting. So it doesn't match the order correctly of the folder